lib/erb_lint/linters/instance_variable.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ERBLint
+  module Linters
+    # Checks for instance variables in partials.
+    class InstanceVariable < Linter
+      include LinterRegistry
+
+      class ConfigSchema < LinterConfig
+        property :partials_only, accepts: [true, false], default: false, reader: :partials_only?
```

Comment:
```suggestion
        property :partials_only, accepts: [true, false], default: true, reader: :partials_only?
```

Instance variables in templates is the Rails default and those linters should follow Rails defaults.